pajbot/models/emote.py

```diff
@@ -12,15 +12,21 @@ class Emote:
     :ivar id: Provider-specific ID, e.g. the emote ID for twitch, or the has for bttv.
     :type id: str
     :var urls: Dict mapping size (e.g. "1", "2", or "4") to an URL depicting the emote
-    :type urls: dict[str, str]"""
+    :type urls: dict[str, str]
+    :ivar max_width: The maximum width the emote can have.
+    :type max_width: int
+    :ivar max_height: The maximum height the emote can have.
+    :type max_height: int"""
```

Comment:
The maximum N of the largest scale emote? something like that is probably clearer